python/partials/_deploy.erb

```diff
@@ -0,0 +1,40 @@
+And with that you can deploy the app!
+
+```cmd
+fly launch
```

Comment:
Do you deploy only with `fly launch`? I think almost all the frameworks have `SkipDeploy: true` ([ref](https://github.com/superfly/flyctl/blob/master/scanner/flask.go#L33))